Cart/cart.css

```diff
@@ -0,0 +1,100 @@
+@import url("/style.css");
+.address-container {
+    border: 1px solid;
+    width: 80%;
+    margin: 0 auto;
+    height: auto;
+}
+
+.btn-address-add {
+    font-size: 1.5rem;
+    color: var(--accent-color);
+}
+
+.card-container {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+}
+
+.ecom-h-card {
+    display: flex;
+    background-color: var(--accent-color);
+    border-radius: 5px;
+}
+
+.ecom-h-img {
+    width: 27rem;
+    height: 100%;
+    border-radius: 5px 0px 0px 5px;
+}
+
+.card-heading {
+    font-size: 3rem;
+}
+
+.card-btn {
+    width: 4rem;
+    height: 1rem;
+}
+
+.ecom-quant {
+    width: 30px;
```

Comment:
be consistent using units. some times for width yu uses px and some time rem.

Reply:
Yeah, here I have used pixels as I dont want to change the width when the screen is resized or viewed on different devices. Anyway, Thanks for the suggestion. I will be consistent using units for lengths